lib/ProductOpener/Display.pm

```diff
@@ -6256,8 +6274,8 @@ HTML
 ;
 	}		
 	
-	
-	if (($lc eq 'fr') and (has_tag($product_ref, "labels","fr:produits-retires-du-marche-lors-du-scandale-lactalis-de-decembre-2017"))) {
+	# Commenting out out of date warnings below
```

Comment:
It's kind of sleeping code instead of dead code: we might revive it when the next warning comes. Usually in situations like that we want to be able to add a warning very quickly without taking too much time to think how/where to add it.

Granted it's not a very good code in the first place. Maybe we could open a specific issue for it. It could be an easy issue to give to potential new coders. What do you think? I was thinking that we could try to list again the "easy" perl issues, and try to post them on the Perl mailing lists to see if we can get more help. :)